### PR TITLE
update aishell-1 recipe with k2.rnnt_loss

### PR DESCRIPTION
**Pull request by pingfengluo**
update aishell-1 recipe with k2.rnnt_loss and a better result (5.7 -> 5.4 ) with same number epochs compare to torchaudio.functional.rnnt_loss.
@csukuangfj  ready for review, thanks.

### CODE REVIEW

**Comment by csukuangfj** [comment]
Thanks! Is there a link to the pre-trained model so that others can download it?

**Comment by csukuangfj** [comment]
Thanks

File: egs/aishell/ASR/transducer_stateless/model.py [hunk 1]
```diff
@@ -108,18 +108,18 @@ def forward(
...
-            targets=y_padded,
-            logit_lengths=x_lens,
-            target_lengths=y_lens,
-            blank=blank_id,
-            reduction="sum",
-        )
+        loss = k2.rnnt_loss(logits, y_padded, blank_id, boundary)
 
-        return loss
+        return torch.sum(loss)
```
Comment (pkufool) [inline egs/aishell/ASR/transducer_stateless/model.py:125]: Just to make you know that, the default reduction for `k2.rnnt_loss` is "mean" (the same as pytorch rnnt_loss), so you don't need a `torch.sum` here.

Comment (danpovey) [reply egs/aishell/ASR/transducer_stateless/model.py:125]: I thought we were summing the loss??
If not, surely the normalization by "frames" (which we do in the diagnostics) would be wrong.

Comment (pkufool) [reply egs/aishell/ASR/transducer_stateless/model.py:125]: We used reduction=="sum" in our expts.

Comment (csukuangfj) [reply egs/aishell/ASR/transducer_stateless/model.py:125]: The returned `loss` is a scalar if `reduction=sum` is used, so we should change `return torch.sum(loss)` to `return loss`, even though `return torch.sum(loss)` is equivalent to `return loss` when `loss` is a scalar.

Comment (pingfengluo) [reply egs/aishell/ASR/transducer_stateless/model.py:125]: Yes,  torch.sum is unnecessary here, will fix it. thanks
